fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/directory/FDBDirectoryTest.java

```diff
@@ -161,7 +161,10 @@ public void testListAll() {
         assertCorrectMetricCount(LuceneEvents.Events.LUCENE_LIST_ALL, 2);
         // Assert that the cache is loaded only once even though directory::listAll is called twice
         assertCorrectMetricCount(LuceneEvents.Events.LUCENE_LOAD_FILE_CACHE, 1);
-        directory.getCallerContext().ensureActive().cancel();
+
+        FDBRecordContext originalContext = directory.getCallerContext();
+        originalContext.ensureActive().cancel();
+        originalContext.close();
```

Comment:
The close here should be unnecessary right, as it is closed in the `tearDown` of the base class?

Reply:
Yeah, I suppose so